packages/editor/src/components/inputs/StringInput.tsx

```diff
@@ -33,16 +33,27 @@ const inputStyle = {
   margin: 0
 }
 
-interface StyledNumericInputProps {
+interface StyledStringInputProps {
   className?: string
-  onChange?: any
+  onChange?: (e: any) => void
```

Comment:
Is there a reason this needs to be `any`? I'd suggest we stick to stricter types.

Reply:
the e value?

Reply:
yes

Reply:
fixed